Experiments/GenerateTestsForResultsetToTableDataTypes.sql

```diff
@@ -0,0 +1,85 @@
+/*
+   Copyright 2011 tSQLt
```

Comment:
@JonathanHallKJR , this all looks great. We'd like to make some small modifications, but critically, we can't use copyrighted code in an open-source framework. Are you ok with us deleting the copyright?

Reply:
@lizbaron 

This file was largely copied from a file in the project, as such, I preserved the license agreement that was in place.

The original file can be found here [/Experiments/GenerateTestsForResultsetFilterDataTypes.sql](https://github.com/tSQLt-org/tSQLt/blob/main/Experiments/GenerateTestsForResultsetFilterDataTypes.sql)